src/utils/cookies.ts

```diff
@@ -0,0 +1,29 @@
+type CookiesOptions = Record<string, string | number | Date | boolean>;
+
+export const setCookie = (name: string, value: string | number | boolean, defaults: CookiesOptions = {}) => {
+    const options: CookiesOptions = {
+        path: '/',
+        ...defaults,
+    };
+
+    if (options.expires instanceof Date) {
+        options.expires = options.expires.toUTCString();
```

Comment:
`options.expires` is too specific for common utility function. Should probably be replaced with `instanceof` check inside `map`

Reply:
fixed